src/main/java/vn/triplet/controller/web/UserController.java

```diff
@@ -29,42 +29,44 @@
 	public String redering(Model model) {
 		return "views/web/profile";
 	}
-	
+
 	@RequestMapping("/welcome")
 	public String welcome(@RequestParam("email") String email, @RequestParam("password") String password,
-			HttpServletRequest request, Model model,final RedirectAttributes redirectAttributes ){
-		User user= userService.findByEmailAndPassword(email, password.trim());
-		if(user!=null)
-		{
+			HttpServletRequest request, Model model, final RedirectAttributes redirectAttributes) {
+		User user = userService.findByEmailAndPassword(email, password.trim());
+		if (user != null) {
 			HttpSession session = request.getSession();
-			redirectAttributes.addFlashAttribute("msg",user.getName().toUpperCase());
+			redirectAttributes.addFlashAttribute("loginsuccess", "WELCOME TO TRIPLET CLOTHES!!");
```

Comment:
message thì khai báo vào file message.properties nhé ` "WELCOME TO TRIPLET CLOTHES!!")